google-cloud-spanner/src/test/java/com/google/cloud/spanner/ValueTest.java

```diff
@@ -85,10 +83,12 @@ public void boolWrapperNull() {
     assertThat(v.getType()).isEqualTo(Type.bool());
     assertThat(v.isNull()).isTrue();
     assertThat(v.toString()).isEqualTo(NULL_STRING);
-
-    expectedException.expect(IllegalStateException.class);
-    expectedException.expectMessage("null value");
-    v.getBool();
+    try {
+      v.getBool();
+      fail("");
```

Comment:
nit: Here and below, please add a text like "Expected exception"

Reply:
Done.